src/ExtensionRoot.ts

```diff
@@ -6,7 +6,7 @@
 import { Guid } from 'guid-typescript';
 import { Telemetry } from 'telaug';
 import * as vscode from 'vscode';
-import TelemetryReporter from 'vscode-extension-telemetry';
+import TelemetryReporter from '@vscode/extension-telemetry';
```

Comment:
💡Thanks for this, Is this working end-to-end? i.e. we have tested data collection? **or** is this just package update, if so lets make sure new telemetry is hooked : https://code.visualstudio.com/api/extension-guides/telemetry Thank you.

Reply:
@Vidya2606 you can generate a VSIX and we can test it along with china testing team to produce enough data to confirm your changes are working. Please let me know for any further questions. Thanks.

Reply:
Working on adding the latest version, once it is resolved I will generate the vsix and send it for testing.